### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by mimeosd**
I believe I have solved most of your issues you had open so examine it and what you like - you can close.

### CODE REVIEW

**Comment by vijay-jaisankar** [comment]
closes: #22 #18 #17 #15

**Comment by vijay-jaisankar** [comment]
Thanks for the contribution @mimeosd ! 
The changes LGTM, take a look at the small review comment, once that is taken care of we can merge this PR 🚀

**Comment by mimeosd** [comment]
I can not locate your review comment...

File: app.py [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
+@app.route('/sitemap.xml')
+def sitemap_xml():
+    return send_from_directory(app.static_folder, 'sitemap.xml')
+
 if __name__ == "__main__":
-    app.run(debug = True)
\ No newline at end of file
+    app.run(debug=True)
+
+# What`s done?
```
[the remark this answers is not in the document]
Comment (vijay-jaisankar) [reply app.py]: cc @mimeosd

**Comment by vijay-jaisankar** [comment]
No worries, I've tagged you in the comment so you should get a Github notification leading you to the comment.

---

In any case, the comment reads:
```
To separate concerns, please remove the Changelog (comments) from the application source and instead add them to the https://github.com/vijay-jaisankar/pandemonium/pull/23.
```

This is regarding the comments present in lines `30` and beyond in `app.py`

**Review by mimeosd — COMMENTED** [review]
What`s done?
1. Removed global statements since its bad practice to rely on constant that some other function in future might modify
2. get_meme_url Result Check - used or to handle None responses more concisely.
3. Simplified the Logic: Removed unnecessary checks or redundant variable assignments.
4. Added more meta tags as per open issue on GH page
5. Removed styles.css - you dont need it since we are inheriting from bootstrap`s css. Unless you specifically want to style page yourself, you do not need styles.css
6. Added robots.txt and sitemap.xml. Since you have static site there is no need for dynaminc generation of urls (you have only index.html)
Thus when you deploy this website to production make sure to modify both files, eg:
  robots.txt
 Sitemap: https://www.pandemonium-memes.com/sitemap.xml
  sitemap.xml
 <loc>https://www.pandemonium-memes.com/</loc>
Documentation - what do you want to document? Only 1 webpage with 1 function and 1 API call so I think documenting it is redundant...
7. In index.html, in head section modify author and twitter handle to your, I put mine for now.

File: app.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vijay-jaisankar) [reply app.py]: Resolved via https://github.com/vijay-jaisankar/pandemonium/pull/23/commits/3deb3a7b12d4515d1df482ed4f128a778f128a61

**Review by vijay-jaisankar — APPROVED** [review]

**Comment by vijay-jaisankar** [comment]
Great, thanks for the contribution!

**Pull request merged**